### PR TITLE
Fix l 1217 - Contact Form Working

### LINKED ISSUE #1217: Contact Form not working
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

Will work on it ??
 Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1217 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- https://pr-1290-fossasia-susper.surge.sh

### CODE REVIEW

**Comment by sk9331657** [comment]
@simsausaurabh  @praveenojha33  I need a gmail account to send free email  because we don't have free endpoints .

**Comment by sk9331657** [comment]
. @praveenojha33  @simsausaurabh  I require email token to replace my token . Kindly generate Token here https://smtpjs.com/.

**Comment by sk9331657** [comment]
@praveenojha33  It is free available smtp javascript for delivering email . As we don't have backend we need external api.

**Comment by sk9331657** [comment]
@mariobehling  @simsausaurabh  Please review.

**Comment by sk9331657** [comment]
@mariobehling  @simsausaurabh  Please review.

**Comment by sk9331657** [comment]
@praveenojha33  What should be done to this now??

File: src/app/contact/contact.component.ts [hunk 1]
```diff
@@ -74,5 +75,18 @@ export class ContactComponent implements OnInit, DoCheck {
             return false;
         }
     }
+    mail () {
+        if (this.tpnoInput && this.tpnoInput.toString().length >= 10 && this.tpnoInput > 0 && this.contactMessage && this.contactMessage.length >= 100 && window.location.href.split('/')[0] === 'https:' ) {
+            let Message = "Name:   " + this.nameInput + "     Contact No:    " + this.countrycode + this.tpnoInput + '       ' + "Message:       " + this.contactMessage;
+            let js1 = document.createElement("script");
+            let ScriptHTML = "Email.send({SecureToken : 'e64d9c8a-8ce6-46f1-8d1c-3b9acebe5c20' ,To:'office@fossasia.org' ,From:'" + this.emailInput + "',Subject:'Contact Form', Body:'" + Message + "'}).then(function(message){if(message === 'OK'){ alert('sent'); } else {alert('Retry'); }});";
```
Comment (simsausaurabh) [inline src/app/contact/contact.component.ts]: Not supposed to use secure tokens like this.

File: src/index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.4.2/css/swiper.min.css">
   <link rel="manifest" href="manifest.json">
   <meta name="theme-color" content="#0d0d63">
+  <script type="text/javascript" src="https://smtpjs.com/v3/smtp.js"></script>
```
Comment (simsausaurabh) [inline src/index.html]: Why using external API?

Comment (sk9331657) [reply src/index.html]: @simsausaurabh  As we know we don't have backend so, I am using smpt.js API which is free to send contact form response. Earlier form was not sending responses .

Comment (sk9331657) [reply src/index.html]: @simsausaurabh  Secure token is used , I cannot my raw gmail login.

Comment (simsausaurabh) [reply src/index.html]: But we are not supposed to display these tokens openly.

Comment (simsausaurabh) [reply src/index.html]: Please also take review from co-developers on Gitter

**Comment by sk9331657** [comment]
@shreyanshdwivedi @AakashMallik @simsausaurabh @praveenojha33 For preventing spam , we will use recaptha and for preventing token misuse we will use environment variables. I will repush the pr by making required changes.

**Comment by praveenojha33** [comment]
@sk9331657 I do not understand what you are trying to do here your issue also is empty and I am unable to know what you are doing. I have already told you that our goal is to reduce external APIs and use YaCy only. I do not think we have a lot of users for which we will introduce recaptcha. Currently our aim is to introduce new features which are possible using YaCy . Thank You

**Comment by sk9331657** [comment]
@praveenojha33  The Contact form in contact section is not working and  not sending responses to office@fossasia.org. This is to fix the form . You can check contact form at susper.com.It doesn't work.

**Comment by praveenojha33** [comment]
@sk9331657 Yeah I know that. But how you will achieve that with YaCy(or any open source API). Please discuss this issue with Mario.

**Comment by simsausaurabh** [comment]
There are conflicts, been a lot of inactivity. Closing this. Displaying tokens openly is not a good idea.

**Pull request closed without merging**